surveys/2021-annual-survey/design/who.md

```diff
@@ -0,0 +1,42 @@
+# Who is using Rust?
+
+At a high-level, there are two groups of information that we want to collect from respondents about who they are:
+* demographic information for cohort analysis
+* demographic information to better understand the overall shape of the community
+
+## Cohort Analysis
+
+We aim to collect demographic information such that we can group survey takers into various cohorts which we can perform cohort analysis which allows for comparing how different cohorts differ across different dimensions. 
+
+Cohort analysis allows us to not only answer questions about the entire dataset of respondents but also subgroups that have potentially interesting defining characteristics.
+
+For example, cohort based on experience with different types of programming languages could allow us to find out if the experience of using Rust is markedly different for systems programmers (i.e., those who exclusively use languages like C and C++) is different than for those who use dynamically-typed languages (i.e., those who exclusively use languages like JavaScript and Python).
+
+### The cohorts 
+
+The cohorts should include:
+
+* Current user of Rust, former user of Rust, non-user of Rust
+* Types of programming languages used (i.e., extensive and current use of a programming language that fits into a given category):
+  * Systems programming (C, C++) 
+  * Statically-typed "modern" language (Haskell, Scala, Swift, Kotlin, etc.)
+  * OO Languages (Java, Objective-C, C#)
+  * Dynamically-typed languages (Ruby, JavaScript, Python)
+* Identifies with an underrepresented group in technology 
+* Industry (e.g., general software, banking, transportation, etc.)
+* Current stack (server networking, desktop application, mobile application, web application, embedded, etc.)
+
+
+## Community shape
+
+Other demographic information can be collected just to better understand the shape of Rust community. While cohort analysis could also be performed based on these factors, we have a strong feeling that these indicators won't lead to interesting cohort analysis. Further investigation may prove this assumption to be incorrect however. 
+
+The other demographic information includes:
+* Location
+* Preferred human language for technical communication 
+
```

Comment:
age, level of industry experience. I could imagine a whole bunch of stuff which could be interesting in a bunch of ways (salary, operating system preference, education level, sex/gender, etc), but I'm not sure how to decide how well these or location/language meet the goals of the survey

Reply:
Location and language were questions we've specifically wanted to answer in the past, but we didn't have the information.

The other information could very much be interesting. We'll need to figure out how we want to balance the amount of information we gather without making respondents uncomfortable or feel like we're collecting too much.